lib/runner.js

```diff
@@ -20,6 +23,7 @@ var chainableMethods = {
 		before: {type: 'before'},
 		after: {type: 'after'},
 		skip: {skipped: true},
+		todo: {skipped: true, todo: true},
```

Comment:
Up until now I've been treating todo as a skipped test, that has failed, mainly based on the TAP spec. Kind of depends on how tightly integrated the runner implementation is with the reporter output.